### PR TITLE
[ADD] sale order related fields

### PR DESCRIPTION
**Pull request by robinkeunen**
https://gestion.coopiteasy.be/web#id=1360&view_type=form&model=project.task&action=479

- [x] add related fields
- [x] add in report
- [x] remove price fields from view

### CODE REVIEW

File: foodhub_custom/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Foodhub Custom",
+    'version': '9.0.1.0.0',
+    'description': """
+        Foodhub customizations""",
+
+    'author': "Coop IT Easy SCRL fs",
+    'website': "http://www.yourcompany.com",
+
```
Comment (houssine78) [inline foodhub_custom/__openerp__.py]: replace by www.coopiteasy.be

File: foodhub_custom/models/stock.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            sale_order = (
+                self.env['sale.order']
+                    .search([('name', '=', origin)])
+            )
+            order_line = (
+                sale_order.order_line
+                          .filtered(
+                            lambda line: line.product_id == move.product_id)
+            )
+            move.sale_line_id = order_line
```
Comment (houssine78) [inline foodhub_custom/models/stock.py]: move.sale_line_id = order_line.id ?

File: foodhub_custom/reports/report_deliveryslip.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            </xpath>
+
+            <xpath expr="//table[@t-if='o.pack_operation_ids']/thead/tr/th[1]" position="before">
+                <th><strong>Price</strong></th>
+                <th><strong>Taxes</strong></th>
+                <th><strong>Total</strong></th>
+            </xpath>
+
+            <xpath expr="//table[@t-if='o.pack_operation_ids']/tbody/tr/td[1]" position="after">
+                <!--in <tr t-foreach="o.pack_operation_ids" t-as="pack_operation">-->
```
Comment (houssine78) [inline foodhub_custom/reports/report_deliveryslip.xml]: remove commented line

**Review by houssine78 — COMMENTED** [review]
ok to merge after taken into account my comment

File: foodhub_custom/models/stock.py (hunk 2 above)
Comment (robinkeunen) [reply foodhub_custom/models/stock.py]: Boh, on manipule des objets ici non? Ça marche en tout cas...

**Pull request merged**